tests/test_cache_invalidation.py

```
@@ -0,0 +1,305 @@
+"""
+Testes específicos para invalidação de cache e geração de chaves avançada.
+"""
+
+import pytest
+import tempfile
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)